satpy/etc/composites/visir.yaml

```diff
@@ -419,12 +419,14 @@ composites:
   _night_background:
     compositor: !!python/name:satpy.composites.StaticImageCompositor
     standard_name: night_background
+    sensor: 'viirs'
```

Comment:
That doesn't seem right.  Why VIIRS?

Reply:
The night background image is derived from VIIRS DNB data.

Reply:
I guess an alternative would be to set the sensor to `generic` or something like that, but I figure we might as well set it to the correct thing...